modules/pagerduty-configuring-acs.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-pagerduty.adoc
+:_module-type: PROCEDURE
+[id="pagerduty-configuring-acs_{context}"]
+= Configuring {product-title}
+
+Create a new integration in {product-title} by using the integration key.
+
+.Procedure
+. On the RHACS portal, navigate to *Platform Configuration* -> *Integrations*.
+. Scroll down to the *Notifier Integrations* section and select *PagerDuty*.
+. Click *New Integration* (add icon).
```

Comment:
Can you use the same formatting for the icons here that you used in the Slack example? I think it was `(`**add**` icon)`